docusaurus/docs/learn/quickstarts/zac/index.md

```diff
@@ -1,122 +1,60 @@
 ---
 title: Ziti Admin Console
+sidebar_label: Console
 ---
 
 import Wizardly from '@site/src/components/Wizardly';
 
 The Ziti Administration Console (ZAC) is a web UI provided by the OpenZiti project which will allow you to configure and
 explore a [network](/learn/introduction/index.mdx).
 
-## Prerequisites
+## Overview
 
-* It's expected that you're using `bash` for these commands. If you're using Windows we strongly recommend that you install
-  and use Windows Subsystem for Linux (WSL). Other operating systems it's recommended you use `bash` unless you are able to
-  translate to your shell accordingly.
+ These steps enable the console for a quickstart controller. Run these commands with a shell like `bash` in a Windows Subsystem for Linux (WSL), macOS, or Linux.
 
-* You will need `node` and `npm` executables from Node.js v16+. If you see an error like the one shown below you likely
-  have a version of node < v16. __You need node v16+__:
+## Downloading From GitHub
 
-      ziti = await loadModule('@openziti/ziti-sdk-nodejs')
-      ^^^^^
-
-      SyntaxError: Unexpected reserved word
-       at Loader.moduleStrategy (internal/modules/esm/translators.js:133:18)
-       at async link (internal/modules/esm/module_job.js:42:21)
+These steps are applicable to both the [Local - No Docker](/learn/quickstarts/network/local-no-docker.md) and the
+[hosted yourself](/learn/quickstarts/network/hosted.md) deployments.
 
- * You will also need the Angular CLI installed and available on your command line. If you do not have this already you can
-   run the following command:
+1. On the controller host, download the latest release of the console from GitHub. You can use any console version >= 3.0.0.
 
     ```text
-    npm install -g @angular/cli@16
+    wget https://github.com/openziti/ziti-console/releases/latest/download/ziti-console.zip
```

Comment:
the quickstart console steps are similiar to the prod Linux console steps, but here are simplified and tailored specifically for the quickstart